uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-calendar.js

```diff
@@ -509,7 +507,31 @@ define([
         }
       });
 
-      return calendar;
+      return indexedByMonth;
+    }
+
+    /**
+     * Show month loader for all months initially
+     * then hide each loader on the interval of an offset value
+     */
+    function showMonthLoader() {
+      var offset = 0;
+
+      vm.months.forEach(function (month, index) {
+        //show loader
+        vm.loading.months[index] = (index !== vm.months.indexOf(vm.selectedMonths[0]));
+
+        //do not hide the month which is selected by default i.e. the current month
+        if(index !== vm.months.indexOf(vm.selectedMonths[0])) {
+          (function (index) {
+            $timeout(function () {
+              vm.loading.months[index] = false;
+            }, offset);
+          })(index);
+
+          offset += 500;
```

Comment:
Initially I tried with `2000ms` which seemed fine but that much delay was not necessary, changing it to `500ms` also caused some minor freeze on the loader, where the loader is not spinning when rendering is happening. So `500ms` is optimal.

Moved to variable